### DISCUSSION TITLE
pdb to cif

### DISCUSSION THREAD

**Discussion by lobnaOmar20** [project-gemmi/gemmi#281] [Q&A] [answered]
can gemmi library change a pdb file to a cif one?? 
I have a small pdb file of 114 atoms. 

I only saw in the gemmi documentation a change from cif to pdb with proteins like also this one (https://stackoverflow.com/questions/60168883/converting-segments-of-large-cif-files-to-smaller-pdb-files) whereas, I have a small crystal structure and want to change from the pdb to the cif. 
here is my pdb: 
[data.txt](https://github.com/project-gemmi/gemmi/files/12596746/data.txt)

I have hundreds of pdb files; therefore I need to do it with python. 

I am really sorry if my question is trivial

**Comment by wojdyr** [comment, marked as answer]
Yes, it can convert pdb file to mmCIF file.
In Python, read the pdb file and write the obtained Structure to mmCIF file.
Or use `gemmi convert file.pdb file.cif` from the command line.

**Reply by lobnaOmar20** [reply]
Thank you for your reply. I really appreciate it :)

**Reply by lobnaOmar20** [reply]
I tried to use gemmi from the command line but I had some problems therefore I used openbabel. I thank you again for your quick replies. I see them instantly.
